src/controllers/albumController.js

```diff
@@ -55,6 +106,15 @@ const updatePriceOfAlbums = async (req, res) => {
 
 //Fetching an album from the database, including all the users that have purchased it
 const getAlbumWithAllUsersWhoPurchasedIt = async (req, res) => {
+  /*
+     #swagger.summary = 'Show all users that purchased this particular album'
+
+     #swagger.responses[200] = {
+				description: 'Album with users who purchased it was fetched  successfully ',
+				chema: { albums: [{ $ref: '#/definitions/Album' }] }
```

Comment:
Probably need to define a schema definition in `swagger.js` for `AlbumWithUsers` so that we can display an example of what this response would look like

Reply:
@akosasante just did it- but can't test it in postman - i log in, create an order and then do GET request to {{MUSICURL}}/albums/64f0777055b548585c
![Captura de pantalla 2023-08-31 a las 13 21 30](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/b2f514ae-f63d-447d-8cb4-69f3368e1692)
a3c18c
![Captura de pantalla 2023-08-31 a las 13 21 39](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/c444d44b-9b70-471c-ba5c-347390907984)
/listOfUsersWhoPurchasedThisAlbum and get the error that there's no such album